plugins/basic/startup/python2.7libs/pythonrc.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2017 Shotgun Software Inc.
```

Comment:
You'll notice that each plugin has 2 identical `pythonrc.py` files in order to support python `2.6` and `2.7`. I tried to keep the code to a minimum in these files and use shared code in `plugin_root/python`. Let me know if you think we should reduce further.